services/blockchain-service/blockchain-service-base.js

```diff
@@ -1204,4 +1206,37 @@ export default class BlockchainServiceBase {
     convertToWei(ether) {
         return Web3.utils.toWei(ether.toString(), 'ether');
     }
+
+    //Paymaster functions
+    async deployPaymasterContractFunction(blockchain, hubAddress) {
```

Comment:
This should be renamed to `deployPaymasterContract`

Reply:
Also all other functions added in this file don't need, don't need to have function in their name